protocol/avdl/chat1/remote.avdl

```diff
@@ -209,4 +213,35 @@ protocol remote {
   // Pubsub endpoints
   void publishReadMessage(gregor1.UID uid, ConversationID convID, MessageID msgID);
   void publishSetConversationStatus(gregor1.UID uid, ConversationID convID, ConversationStatus status);
+
+  SetSettingsRes setSettings(
+    // null means the setting applies to all conversations.
+    union { null, ConversationID } convID,
+    // null means the setting applies to all devices.
+    union { null, gregor1.DeviceID } deviceID,
+    array<SettingKV> settings
+  );
```

Comment:
The null values aren't implemented, but I think we might want them there in the future.